package.json

```diff
@@ -44,8 +44,8 @@
   "dependencies": {
     "@slack/logger": "^2.0.0",
     "@slack/oauth": "^1.2.0",
-    "@slack/types": "^1.6.0",
-    "@slack/web-api": "^5.9.0",
+    "@slack/types": "feat-workflow-steps",
+    "@slack/web-api": "feat-workflow-steps",
```

Comment:
Similarly, need to update the versions of these to real versions once they are released